### PR TITLE
[Dev] Surface Magnetism + Spatial Processing example scene - improvement

### PR DESCRIPTION
**Pull request by cre8ivepark**
Overview
---
- Added transparent surface to make it easier to see the interaction with physical surfaces.
- Assigned WireframeBlue material to the example scene. (it was not assigned)
- [Updated landing page README for the discoverability](https://github.com/cre8ivepark/MixedRealityToolkit-Unity/blob/Dev_Working_Branch_SurfaceMagnetism/README.md)
2018-05-17 13_28_16-mixedrealitytoolkit-unity_readme md at dev_working_branch_surfacemagnetism cre

Changes
---
- Updates to #2113

### CODE REVIEW

File: ProjectSettings/TagManager.asset [hunk 1]
```diff
@@ -13,7 +13,7 @@ TagManager:
   - UI
   - 
   - 
-  - 
+  - SpatialProcessingPlane
```
Comment (StephenHodgson) [inline ProjectSettings/TagManager.asset]: Not sure we should be adding anything to the tag manager manually. 

When people import this example, tags are generally not included.

Comment (cre8ivepark) [reply ProjectSettings/TagManager.asset]: If that's the case, I guess developers have to manually add the layer?

Comment (StephenHodgson) [reply ProjectSettings/TagManager.asset]: indeed, or we can hard code it.

Comment (StephenHodgson) [reply ProjectSettings/TagManager.asset]: It's related to #1443

Comment (cre8ivepark) [reply ProjectSettings/TagManager.asset]: I just read #1443. In this case, I think we can use layer 31:Spatial Mapping and add instruction to use Configure > Apply Mixed Reality Project Settings.
Thank you!

2018-05-17 12_49_47-unity 2017 2 1p2 64bit - spatialprocessingsurfacemagnetism unity - mrtk - univ

2018-05-17 12_49_40-unity 2017 2 1p2 64bit - spatialprocessingsurfacemagnetism unity - mrtk - univ

**Comment by cre8ivepark** [comment]
- Removed TagManager.asset
- Updated README to use MRTK's Configure menu to add the layer 31:SpatialMapping
- Updated scene to use the layer 31:SpatialMapping

https://github.com/cre8ivepark/MixedRealityToolkit-Unity/blob/may18_dev_SurfaceMagnetism/Assets/HoloToolkit-Examples/SpatialMapping/SpatialProcessingSurfaceMagnetism.md

File: Assets/MixedRealityToolkit-Examples/SpatialMapping/SpatialProcessingSurfaceMagnetism.md [hunk 2]
```diff
@@ -12,17 +12,11 @@ https://gfycat.com/PitifulRareKodiakbear
...
-<img src="/External/ReadMeImages/MRTK_SurfaceMagnetism1.png" width="550">
-<img src="/External/ReadMeImages/MRTK_SurfaceMagnetism2.png" width="300">
-<img src="/External/ReadMeImages/MRTK_SurfaceMagnetism3.png" width="550">
-<img src="/External/ReadMeImages/MRTK_SurfaceMagnetism4.png" width="300">
+1. Add a new layer for the planes created by Spatial Processing. Mixed Reality Toolkit's **Apply Mixed Reality Projcet Settings** automatically addes a new layer **31:Spatial Mapping**.<img src="/External/ReadMeImages/MRTK_SurfaceMagnetism1.png" width="550"> <img src="/External/ReadMeImages/MRTK_SurfaceMagnetism2.png" width="300"> <img src="/External/ReadMeImages/MRTK_SurfaceMagnetism3.png" width="550"> <img src="/External/ReadMeImages/MRTK_SurfaceMagnetism4.png" width="300">
 
-2. In SpatialMapping prefab, make sure to set the Physics Layer to 31.
-<img src="/External/ReadMeImages/MRTK_SurfaceMagnetism5.png" width="550">
+2. In SpatialMapping prefab, make sure to set the Physics Layer to 31. <br/><img src="/External/ReadMeImages/MRTK_SurfaceMagnetism5.png" width="550">
```
Comment (davidkline-ms) [inline Assets/MixedRealityToolkit-Examples/SpatialMapping/SpatialProcessingSurfaceMagnetism.md]: This is the default within the prefab. I would reword step 2 to say "make sure the Physics Layer is set to 31"

File: Assets/MixedRealityToolkit-Examples/SpatialMapping/SpatialProcessingSurfaceMagnetism.md [hunk 3]
```diff
@@ -12,17 +12,11 @@ https://gfycat.com/PitifulRareKodiakbear
...
 
-3. Assign SolverSurfaceMagnetism script to an object. In Magnetic Surface option, select the new layer **SpatialMapping** that you created. 
-<img src="/External/ReadMeImages/MRTK_SurfaceMagnetism6.png" width="550">
+3. Assign SolverSurfaceMagnetism script to an object. In Magnetic Surface option, select the new layer **SpatialMapping** that you created. <br/><img src="/External/ReadMeImages/MRTK_SurfaceMagnetism6.png" width="550">
```
Comment (davidkline-ms) [inline Assets/MixedRealityToolkit-Examples/SpatialMapping/SpatialProcessingSurfaceMagnetism.md]: "select the SpatialMapping layer."  they would not have created it

Comment (cre8ivepark) [reply Assets/MixedRealityToolkit-Examples/SpatialMapping/SpatialProcessingSurfaceMagnetism.md]: Updated, Thanks!
